docs/platform/howto/add-storage-space.rst

```diff
@@ -24,7 +25,7 @@ You can add storage to your running service in `Aiven Console <https://console.a
 
 #. In the **Upgrade service storage** window, use the slider to add disk storage.
 
-   ..note::
+.. note::
```

Comment:
Thanks for fixing that! Still, now step 5 "Select Save changes" shows as step 1 in the "Add storage to a running service" section. Please change "1" into "5".